modules/apps/account/account-test/src/testFunctional/tests/Account.testcase

```diff
@@ -4050,6 +4050,227 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-188650 (TC-2). Headless Admin User can list the ACTIVE Users on a certain API page sorted by given name"
+	@priority = 4
+	test HeadlessAdminCanListActiveUsersWithPagingAndSortingByGivenName {
+		task ("Given: Multiple Users are created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Users",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			for (var letter : list "a,b,c,d,e,f,g,h,i,j,k,l") {
```

Comment:
Can you formulate a dedicated macro for this loop?

Reply:
Done